modules/nf-core/sequencetools/pileupcaller/main.nf

```diff
@@ -42,4 +43,23 @@ process SEQUENCETOOLS_PILEUPCALLER {
         sequencetools: \$(echo \$(pileupCaller --version 2>&1) )
     END_VERSIONS
     """
+
+    stub:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args_list = args.tokenize()
```

Comment:
We should definitely have some kind of standard in nf-core to check all the parameters.
As for now everyone has its own way but we should find a way to harmonize this.
I really like `args.tokenize()`